### PR TITLE
Try to always have composer selected if possibl

### PR DESCRIPTION
**Pull request by Jikstra**
e, without preventing clicking on other inputs/selecting text

I would wait for merging and include this only in 1.15 as this probably causes new bugs?

### CODE REVIEW

**Comment by Jikstra** [comment]
Lets get it merged, I don't like prs which are getting moldy. We can test this the next days from the master branch.

**Comment by Simon-Laux** [comment]
I would wait until 1.14 is out...

File: src/renderer/components/message/MessageList.tsx [hunk 1]
```diff
@@ -107,6 +107,11 @@ export default function MessageList({
     composerTextarea && composerTextarea.focus()
   }, [chat.id])
 
+  useEffect(() => {
+    const composerTextarea = refComposer.current.childNodes[1]
```
Comment (Simon-Laux) [inline src/renderer/components/message/MessageList.tsx:111]: I don't quite like the usage of `childNodes[1]`, but well maybe there is no obvoius more elegant solution...

**Comment by Simon-Laux** [comment]
double click for quoting doesn't refocus the composer

**Pull request merged**